### ISSUE TITLE
boot into black screen after installation on manjaro linux

### ISSUE THREAD

**Issue by HilariousJon** [prasanthrangan/hyprdots#1767]
# Description

I opened a post on manjaro official forum and describe my situation there. Really hope that someone can help me. 

the post can be refer via this [link](https://forum.manjaro.org/t/system-boot-into-black-screen-after-installing-hyprland/167131/15)

**Comment by kRHYME7** [comment]
Can you run `Hyprland ` in the tty?

Also sddm did not display after boot?

**Comment by HilariousJon** [comment]
Dear @kRHYME7 
oh yes, I can run `Hyprland` in tty, it makes things appear, but how to make it appear automatically, so I don't need tty? sddm runs well as I typed `systemctl status display-manager` it shows that it works well

**Comment by HilariousJon** [comment]
and I notice that I run `code` in terminal, things will crash, how to solve it?

**Comment by kRHYME7** [comment]
systemctl enable --now sddm

**Comment by HilariousJon** [comment]
@kRHYME7 
`systemctl enable --now sddm` didn't work.

**Comment by kRHYME7** [comment]
does sddm show after boot?

**Comment by HilariousJon** [comment]
https://vimeo.com/manage/videos/1001468638 
this video can show my current situation

**Comment by kRHYME7** [comment]
You got different  issues but maybe can be streamlined that you're using an asus laptop. 

problems:
1. sddm did not show up
2. electron apps like 
2.a vscode won't  open
2.b obisidian (according to the logs you got errors rendering them, I'm not sure but maybe something to do with gpu)

I guess you should follow  this first, specially using an Asus laptop https://asus-linux.org/guides/arch-guide/

thankfully we got a script to do the things inside that tutorial, just run `  Hyde inject asus_patch  `  

Take note that I don't know if this could fix your issue, but I'm pretty sure these fix/patch are prerequisites for your arch installation, so it should be good to have it. 


Also, some apps eg vscode got flags to disable gpu rendering you can test it byr running `--disable-gpu ` .

**Comment by HilariousJon** [comment]
@kRHYME7 
Yes, I checked the crash list of the system, and it seems that electron directly crashed in my hyprland desktop env.

and actually I found 

> To ease the installation you can use other distro based on Arch, only Manjaro is highly not recommend (it is not really based on Arch and might not be compatible with points above and things might break with upgrades).

:joy: are you sure it is a problem caused by asus? I found another issue in this repo, whose screenshots shows that it install hyprland on manjaro pretty well.

refer it here: https://github.com/prasanthrangan/hyprdots/issues/558

**Comment by kRHYME7** [comment]
> 
![image](https://github.com/user-attachments/assets/01ca209d-ae21-41dd-8d55-eba730354090)

Looks normal to me and nothing is change here, just the ` pokemon-colorscripts ` command

>  I think it maybe relate to some global configuration problem

this repo generally sets the Sessions envs (because user logs in > hyprland opens with the env vars) 

I guess you should try to fix your sddm errrors first, how about try to install/change a theme using ` Hyde sddm install `
![image](https://github.com/user-attachments/assets/85316818-0e6b-479c-bc7f-bcd6cde35702)

about other rendering stuff are you using nvidia? too ? You check out nvidia page of the hyprland wiki

**Comment by HilariousJon** [comment]
@kRHYME7 
I tried, running `Hyde sddm install`, nothing change. :thinking_face: should have changed if it is caused by missing sddm theme.

I have a nvdia card, I ll go and check hyprland wiki.

---

about the pokemon script, I just don't know how to operate on it :joy: any wiki?

**Comment by thyeun** [comment]
@HilariousJon To solve your ```sddm```, re-install it with ```yay -S sddm-git``` (during install, it will ask you to remove ```sddm```, due to conflict, you just ```Yes``` to remove).

**Comment by HilariousJon** [comment]
@thyeun 
have tried reinstall `sddm`, not working. and the solution for `code` not working as well.

**Comment by thyeun** [comment]
@HilariousJon Your ```code``` is vs code or ```OSS-code```?

**Comment by thyeun** [comment]
@HilariousJon If vs code it should be workable. As you can see my vs code below.

![240822_18h20m28s_screenshot](https://github.com/user-attachments/assets/2bc28d37-6b70-4559-8721-879c24ed90e0)

**Comment by HilariousJon** [comment]
@thyeun 
no idea :joy:, its just super weird. the log shows that it indeed crash. Check the error message above. I also can see that when I press `linux+A` I can see clearly that `visual studio code` is in my software list. But I cannot open it either by running terminal command nor click on the icon.

**Comment by thyeun** [comment]
@HilariousJon You re-install ```sddm``` or re-install with ```sddm-git```? if you just reinstall ```sddm```, your ```sddm``` still wont work, you need to install ```sddm-git```

**Comment by HilariousJon** [comment]
@thyeun 
I re-install `sddm-git`. But let me try again.

**Comment by thyeun** [comment]
> @thyeun no idea 😂, its just super weird. the log shows that it indeed crash. Check the error message above. I also can see that when I press `linux+A` I can see clearly that `visual studio code` is in my software list. But I cannot open it either by running terminal command nor click on the icon.

Do you do exactly what i mentioned by created the file ```code-flags.conf```?

**Comment by thyeun** [comment]
> @thyeun I re-install `sddm-git`. But let me try again, as I can't really remember some of the error message.

Remember to reboot after install, not just login/logout

**Comment by HilariousJon** [comment]
> I re-install sddm-git. But let me try again, as I can't really remember some of the error message.
Yes I have rebooted.

**Comment by HilariousJon** [comment]
> Do you do exactly what i mentioned by created the file code-flags.conf?

yes, I have. the following are current content:

```
# --ozone-platform-hint=wayland
# --disable-gpu
--ozone-platform-hint=wayland
--use-gl=egl
```

**Comment by thyeun** [comment]
@HilariousJon ```sudo pacman -Qs nvidia```, what is your output? if you install correctly as below, everything should work out of the box

![240822_18h39m30s_screenshot](https://github.com/user-attachments/assets/a82618b3-45b2-4743-aa5c-552396792b8b)

**Comment by HilariousJon** [comment]
here are my outputs for `sudo pacman -Qs nvidia`, I guess no particular error?

```
❯ sudo pacman -Qs nvidia
local/egl-wayland 4:1.1.13.1-1
    EGLStream-based Wayland external platform
local/ffnvcodec-headers 12.2.72.0-1
    FFmpeg version of headers required to interface with Nvidias codec APIs
local/lib32-libvdpau 1.5-2
    Nvidia VDPAU library
local/lib32-nvidia-utils 550.107.02-1
    NVIDIA drivers utilities (32-bit)
local/lib32-vulkan-nouveau 1:24.1.6-1
    Open-source Vulkan driver for Nvidia GPUs - 32-bit
local/libvdpau 1.5-3
    Nvidia VDPAU library
local/libxnvctrl 550.107.02-1
    NVIDIA NV-CONTROL X extension
local/mhwd-nvidia 550.107.02-1
    MHWD module-ids for nvidia 550.107.02
local/mhwd-nvidia-390xx 390.157-14
    MHWD module-ids for nvidia 390.157
local/mhwd-nvidia-470xx 470.256.02-4
    MHWD module-ids for nvidia 470.256.02
local/nvidia-dkms 550.107.02-1
    NVIDIA drivers - module sources
local/nvidia-settings 550.107.02-1
    Tool for configuring the NVIDIA graphics driver
local/nvidia-utils 550.107.02-1
    NVIDIA drivers utilities
local/vulkan-nouveau 1:24.1.6-1
    Open-source Vulkan driver for Nvidia GPUs
local/xf86-video-nouveau 1.0.17-3 (xorg-drivers)
    Open Source 3D acceleration driver for nVidia cards
```

**Comment by thyeun** [comment]
@HilariousJon Since you have mix nvidia driver, change the below to and remove ```--use-gl=egl```.

```
--ozone-platform-hint=default
```

**Comment by HilariousJon** [comment]
@thyeun 

after the change, I have: 

```
❯ source ~/.config/code-flags.conf
zsh: command not found: --ozone-platform-hint=default
❯ code
Warning: 'ozone-platform-hint' is not in the list of known options, but still passed to Electron/Chromium.
```

**Comment by thyeun** [comment]
@HilariousJon ```code``` not open? if previous with ```wayland```, ```code``` is open, than it mean workable, the message is just a warning.

**Comment by HilariousJon** [comment]
@thyeun 

it hasn't. haven't opened so far.

**Comment by thyeun** [comment]
@HilariousJon My advised is, dont using this dotfile, since your system abit messy, we dont know which and which driver are useful to your current system. Best use for this dotfile is fresh installation.

**Comment by HilariousJon** [comment]
@thyeun 

Could you be more specific? What dotfile? Maybe I can do a timeshift and try to deal with those drivers.

**Comment by thyeun** [comment]
Hyprdots is a dotfile

**Comment by HilariousJon** [comment]
but i guess currently it is hard to revert it to original kde.

**Comment by HilariousJon** [comment]
It seems that all electron related apps are not able to run. It should be a general problem.

**Comment by thyeun** [comment]
> It seems that all electron related apps are not able to run. It should be a general problem.

For sure it is only happen to you. And it is not general problem, all my electron apps working according what i want, and it done well in my system (nvidia + wayland + hyrpland + Hyprdots).

Why you feel hard to revert back? since you have timeshift.

**Comment by HilariousJon** [comment]
thats the problem :joy: I didn't safe a timeshift..... I wasn't that familiar with the system, since what I took kde as a tool of startiing vscode :joy:

**Comment by thyeun** [comment]
@HilariousJon Few things need you to be verify.

1. Your KDE is wayland or x11?
2. Previous SDDM in KDE working fine?

I might possible guide you to get back your KDE, if you have both info above

**Comment by HilariousJon** [comment]
Hi @thyeun 

1. kde is x11
2. previous sddm works fine within kde

**Comment by thyeun** [comment]
@HilariousJon Good info, so check below.

1. Go ```/etc/sddm.conf.d/```, check how many files inside?

Best if you can show a screenshot.

**Comment by HilariousJon** [comment]
```
❯ ls -al /etc/sddm.conf.d/
.rw-r--r--  23 root  2 5月  05:09  00_manjaro_settings.conf
.rw-r--r-- 199 root 22 8月  20:06  kde_settings.conf
.rw-r--r-- 199 root 22 8月  20:06  kde_settings.t2.bkp
.rw-r--r--  31 root  2 5月  05:09  virtualkeyboard.conf
```

**Comment by thyeun** [comment]
@HilariousJon Keep ```00_manjaro_settings.conf``` and ```05:09  virtualkeyboard.conf```, copy both ```kde``` to your any local folder, afte that remove it.

Than reboot (i think from here, you know what problem), you have two config file

**Comment by HilariousJon** [comment]
done, then?

**Comment by thyeun** [comment]
> done, then?

reboot and see the outcome

**Comment by HilariousJon** [comment]
yes have reboot, no significant outcome :joy:

**Comment by thyeun** [comment]
@HilariousJon i cant see what you have, if you just mentioned no significant outcome, i will just assume your system die, than i cant help you at all. You have to fix it yourself.

**Comment by HilariousJon** [comment]
hmm, it just same outcome as the video I shown. Nothing change.
> https://vimeo.com/manage/videos/1001468638

**Comment by kRHYME7** [comment]
Try to cat this it ahould contain the older sddm config

kde_settings.t2.bkp

**Comment by HilariousJon** [comment]
```
❯ cat ~/Desktop/sddm_tmp/kde_settings.t2.bkp
[Autologin]
Relogin=false
Session=
User=

[General]
HaltCommand=/usr/bin/systemctl poweroff
RebootCommand=/usr/bin/systemctl reboot

[Theme]
Current=Corners

[Users]
MaximumUid=60513
MinimumUid=1000
```

**Comment by kRHYME7** [comment]
> has it overwritten by this command?
Hyde sddm install


nope, by ./install.sh initially. that's why it is good to use this dotfiles in a minimal arch install

**Comment by thyeun** [comment]
@kRHYME7 it is same file as ```00_manjaro_settings.conf``` that why i asked him keep it.

@HilariousJon check here see anything extra file here ```/usr/lib/sddm/sddm.conf.d/```

**Comment by HilariousJon** [comment]
it attains a `default.conf` here

**Comment by kRHYME7** [comment]
hey we got a discord server also a discussion tab here. Let's move this so we won't clutter the issue tab
